tests/bson/bson-int64-compare-001.phpt

```diff
@@ -0,0 +1,34 @@
+--TEST--
+MongoDB\BSON\Int64 comparisons
+--FILE--
+<?php
+
+$max = unserialize('C:18:"MongoDB\BSON\Int64":43:{a:1:{s:3:"int";s:19:"9223372036854775807";}}');
+$min = unserialize('C:18:"MongoDB\BSON\Int64":44:{a:1:{s:3:"int";s:20:"-9223372036854775808";}}');
+$zero = unserialize('C:18:"MongoDB\BSON\Int64":24:{a:1:{s:3:"int";s:1:"0";}}');
```

Comment:
How do comparisons between normal ints and this Int64 class work?

Reply:
I didn't look into that. Do compare handlers facilitate comparisons against scalar types? If so, I'm happy to research that today (any links to prior art would be appreciated), add it, and create some tests.

Reply:
I've opened [PHPC-1213](https://jira.mongodb.org/browse/PHPC-1213) and [PHPC-1214](https://jira.mongodb.org/browse/PHPC-1214) to track implementation of compare and do_operation handlers.